pkgs/by-name/le/leftwm-theme/package.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchFromGitHub, rustPlatform, openssl, pkg-config }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "leftwm-theme";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "leftwm";
+    repo = pname;
```

Comment:
```suggestion
    repo = "leftwm-theme";
```

Reply:
Indeed unnecessary usage of variables, thanks for the suggestion!

Reply:
- https://github.com/NixOS/nixpkgs/issues/277994

Reply:
Woah, had no idea! Will keep a note of this, my thanks